app/jobs/dashboard/applicant_email_job.rb

```diff
@@ -2,9 +2,13 @@ module Dashboard
   class ApplicantEmailJob < ActiveJob::Base
     include SuspendableJob
 
+    APPLY_EMAIL_REGEX = /#{Rails.configuration.x.applyemail.suffix}/.freeze
```

Comment:
Can we name this to email_domain?  We are (hopefully!) blocking by the digital.justice domain not necessarily the account.